api-server/src/homepage/index.html

```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8"/>
+</head>
+
+<body>
+  <div class="overlord">
+    <div class="welcome-text">
+      Secret Database Dashboard<span class="blink"> &#9610;</span>
+    </div>
+
+    <div class="buttons-container">
+      <a href="https://www.youtube.com/watch?v=dQw4w9WgXcQ">
+        <button type="button" class="death-button">Delete Database</button>
+      </a>
+      <a>
+        <button type="button" class="happy-button" onclick="retreat()">Retreat</button>
+      </a>
+    </div>
+  </div>
+</body>
+
+<script>
+  function retreat() {
+    window.history.back(); // Either backup,
+    window.location.href = "https://weather.codingcorner.club/"; // or get sent to the web app
+  }
+</script>
+
+<style>
+  body {
+    background-color: black;
+  }
+
+
+
+  .welcome-text{
+  color: #00ff00;
+  position: absolute;
+  left: 50%;
+  top: 25%;
+  transform: translate(-50%, -50%);
+  padding: 20px;
+  line-height: 30px;
+
+  font-size: 4em;
+  font-family: 'Courier New', Courier, monospace;
+  white-space: nowrap;
+  overflow: hidden;
+  width: 16em;
+  animation: type 4s steps(60, end); 
+  }
+
+  .welcome-text:nth-child(2){
+    animation: type2 8s steps(60, end);
+  }
+
+  .blink {
+    animation: blink 1s infinite;
+  }
+
+  @keyframes type{ 
+    from { width: 0; } 
+  } 
+
+  @keyframes type2{
+    0%{width: 0;}
+    50%{width: 0;}
+    100%{ width: 100; } 
+  } 
+
+  @keyframes blink{
+    to{opacity: .0;}
+  }
+
+  ::selection{
+    background: black;
+  }
+
+.death-button {
+  -webkit-border-radius: 25;
+  -moz-border-radius: 25;
+  border-radius: 25px;
+  font-family: Courier New;
+  color: #ffffff;
+  font-size: 3em;
+  background: #b30000;
+  padding: 10px 20px 10px 20px;
+  border: solid #000000 2px;
+  text-decoration: none;
+}
+
+.death-button:hover {
+  background: #8c0303;
+  text-decoration: none;
```

Comment:
Text decoration of `none` should be inherited from the base `.death-button` class, no?